Cannot find module **'src/types/index.js**

**Describe the bug**
In new release 0.4.7
```
Error: node_modules/@lyrasearch/lyra/dist/types/facets.d.ts:1:48 - error TS2307: Cannot find module 'src/types' or its corresponding type declarations.

1 import { PropertiesSchema, PropertyType } from "src/types";
                                                 ~~~~~~~~~~~


Error: node_modules/@lyrasearch/lyra/dist/types/filters.d.ts:1:39 - error TS2307: Cannot find module 'src/types/index.js' or its corresponding type declarations.

1 import type { PropertiesSchema } from "src/types/index.js";
```

**To Reproduce**
Steps to reproduce the behavior:
1. npm install
2. npm start

**Additional context**
I fix the proposed error with replacing:
```import { PropertiesSchema, PropertyType } from "src/types";``` to
```import { PropertiesSchema, PropertyType } from "@lyrasearch/lyra/dist/types";``` in **node_modules/@lyrasearch/lyra/dist/types/facets.d.ts**
and
```import { PropertiesSchema } from "src/types";``` to
```import { PropertiesSchema } from "@lyrasearch/lyra/dist/types";```  in **node_modules/@lyrasearch/lyra/dist/types/filters.d.ts**

--- a/src/types/facets.ts
+++ b/src/types/facets.ts
@@ -1,4 +1,4 @@
-import { PropertiesSchema, PropertyType } from "src/types";
+import { PropertiesSchema, PropertyType } from "./index.js";
 
 export type FacetSorting = "asc" | "desc" | "ASC" | "DESC";
 

--- a/src/types/filters.ts
+++ b/src/types/filters.ts
@@ -1,4 +1,4 @@
-import type { PropertiesSchema } from "src/types/index.js";
+import type { PropertiesSchema } from "./index.js";
 
 export type FilterOperation =
   | "gt"

--- a/tests/filters.test.ts
+++ b/tests/filters.test.ts
@@ -283,8 +283,10 @@ t.test("should throw when using multiple operators", async t => {
       term: 'coffee',
       where: {
         rating: {
+          // @ts-expect-error error case
           gt: 4,
-          lte: 5,
+          // @ts-expect-error error case
+          lte: 10
         } 
       }
     });

